### ISSUE TITLE
[BUG] Support deployment to folder

### ISSUE THREAD

**Issue by markdumay** [gethinode/hinode#967]
## Describe the bug

Certain hosting providers such as GitHub pages expect the files to be deployed to a subfolder. Currently, Hinode does not support these folders, resulting in broken links. 

## To reproduce

Include a subfolder in the site's  `baseURL`. For example, set the following value in `hugo.toml` and run a local server with `hugo server`:

```toml
baseURL = "http://localhost:1313/subfolder/"
```

## Expected behavior

Hinode should work regardless of the `baseURL` setting.

## Additional context

See #937

**Comment by vinser** [comment]
And GitLab too as I know

**Comment by markdumay** [comment]
@vinser I have released [v0.24.0-beta8](https://github.com/gethinode/hinode/releases/tag/v0.24.0-beta8) to support deployment to a subfolder. A temporary demo is available on https://markdumay.github.io/hinode_gh_test/ (using this [repo](https://github.com/markdumay/hinode_gh_test)). Feedback is welcome!

**Comment by markdumay** [comment]
[v0.24.0-beta9](https://github.com/gethinode/hinode/releases/tag/v0.24.0-beta9) fixes a rendering issue with the sidebar.

**Comment by vinser** [comment]
It turned out that cool developers edit the code faster than inexperienced ones design a website with 5 pages using this code )))
With gopher replacement in go.mod
```go
replace github.com/gethinode/hinode => github.com/gethinode/hinode v0.24.0-beta9
```
and
```
baseURL = 'https://vinser.github.io/my-hinode-site/'
```
I've clicked logo, menu items, languages, modes, breadcrumbs, next/previous, paginator, socials, external links - all these work. 
Great!

Didn't test sidebar because I do not have doc pages yet.

**Comment by markdumay** [comment]
Great! I'll close this issue as resolved then - do let me know when you find any issues.

**Comment by vinser** [comment]
Sorry, I just found that logo.svg works fine but logo-light.svg and logo-dark.svg are not displayed with subfolder in baseURL
Without subfolder both light and dark are rendered good

**Comment by markdumay** [comment]
Thanks for spotting this - I made some changes to the navbar: https://github.com/gethinode/hinode/releases/tag/v0.24.0-beta10

**Comment by markdumay** [comment]
Released in [v0.24.0](https://github.com/gethinode/hinode/releases/tag/v0.24.0)
